src/python/bot/fuzzers/options.py

```diff
@@ -157,22 +158,59 @@ def get_ubsan_options(self):
     return self._get_option_section('ubsan')
 
 
+def get_fuzz_target_support_file(fuzz_target_path, file_extension):
+  """Helper function. returns a file path for a fuzz target support file
+  that has the given file extension"""
+  file_path = fuzzer_utils.get_supporting_file(fuzz_target_path, file_extension)
+
+  if environment.is_trusted_host():
+    file_path = fuzzer_utils.get_file_from_untrusted_worker(file_path)
+
+  if not os.path.exists(file_path):
+    return None
+
+  return file_path
+
+
 def get_fuzz_target_options(fuzz_target_path):
   """Return a FuzzerOptions for the given target, or None if it does not
   exist."""
-  options_file_path = fuzzer_utils.get_supporting_file(fuzz_target_path,
-                                                       OPTIONS_FILE_EXTENSION)
-  options_cwd = os.path.dirname(options_file_path)
 
-  if environment.is_trusted_host():
-    options_file_path = fuzzer_utils.get_file_from_untrusted_worker(
-        options_file_path)
+  options_file_path = get_fuzz_target_support_file(fuzz_target_path,
+                                                   OPTIONS_FILE_EXTENSION)
 
-  if not os.path.exists(options_file_path):
+  if not options_file_path:
     return None
 
+  options_cwd = os.path.dirname(options_file_path)
+
   try:
     return FuzzerOptions(options_file_path, cwd=options_cwd)
   except FuzzerOptionsException:
     logs.log_error('Invalid options file: %s.' % options_file_path)
     return None
+
+
+def get_fuzz_target_grammar(fuzz_target_path):
+  """Return the grammra for a given target or None if it does not exist"""
```

Comment:
typo in `grammra` and missing trailing period

Reply:
Done